react/components/MiniCartContent.js

```diff
@@ -128,6 +134,27 @@ class MiniCartContent extends Component {
     </div>
   )
 
+  createProductShapeFromItem = item => {
+
+    return {
```

Comment:
In this case, you can remove the `return`.

Reply:
`createProductShapeFromItem = item => ({returnedObject})`

Reply:
done.